src/components/main-page/EventDetailsExtended/EventDetailsExtended.tsx

```diff
@@ -10,45 +10,32 @@ export function EventDetailsExtended() {
         AKAI Camp 2023
       </Heading>
       <Text size="m" className={styles.text}>
-        Z ogromną radością prezentujemy pierwszą edycję konferencji{" "}
-        <strong>AKAI&nbsp;Camp</strong>&nbsp;-&nbsp;kolejny rozdział w
-        inspirującej podróży, którą rozpoczęliśmy podczas Hackathonu AKAI Code.
-        Tym razem skupiamy się wokół tematyki <strong>Rozwoju Kariery</strong>,
-        tworząc wyjątkowe wydarzenie dla wszystkich tych, którzy pragną podnieść
-        wartość swojego CV oraz poszerzyć horyzonty zawodowe. Impreza odbędzie
-        się na <strong>Politechnice Poznańskiej</strong>.
+        Pierwsza edycja konferencji <strong>AKAI&nbsp;Camp</strong>{" "}
+        organizowanej przez AKAI -
+        Akademickie&nbsp;Koło&nbsp;Aplikacji&nbsp;Internetowych - odbędzie się
+        na <strong>Politechnice Poznańskiej</strong>. Ta inicjatywa jest
+        kolejnym rozdziałem inspirującej podróży, którą rozpoczęliśmy podczas
+        Hackathonu AKAI Code. Tym razem skupiamy się wokół tematyki{" "}
+        <strong>Rozwoju Kariery</strong>.
       </Text>
+
+      {
+        //TODO: Czy tekst nie powinien być kierowany do jednej osoby?
```

Comment:
w sensie ze my piszemy w pierwszej osobie czy piszemy do jednego uczestnika?

Reply:
Czasami w tekście zwracamy się do ludzi (wszystkich potencjalnych uczestników), a czasami do jednej osoby (używając np. "Ciebie", "Tobie"). Nie wiem czy takie mieszanie jest dobre, więc może lepiej zdecydować się na jedno.

To też trochę pytanie do dziewczyn 😅